openapi.yaml

```
@@ -810,6 +810,39 @@ paths:
         '422':
           $ref: '#/components/responses/unprocessableEntity'
     patch:
-      summary: 現在自分がログインしているユーザの情報を更新する
+      summary: 提供されたトークンに対応するユーザの情報を更新する
+      operationId: PatchUser
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              description: 必須パラメータ自体は存在しないが、パラメータがない場合はパラメータの不備を理由に400(Bad Request)を返す。
```

少し持って回った言い方に感じました。
前半部分は必要ないのでは?

```suggestion
              description: パラメータが指定されていない場合は400(Bad Request)を返す。
```

---

まあ、ドキュメントを見れば必須パラメータが存在しないことはわかるから不要なのかな。必須パラメータはないけど、少なくとも一つのパラメータは必須ということを表現しようとした結果ですかね。

`required` というフィールドに必須パラメータのキー名を指定するのですが、それの記載がないのは意図しているということを表現したかったというのもあります。